Components/Services/BasketService.php

```diff
@@ -40,11 +42,16 @@ class BasketService
      */
     private $logger;
 
+    /** @var DIContainer */
+    private $container;
+
     /**
      * @param ModelManager $modelManager
      * @param LoggerInterface $logger
+     * @param DIContainer $container
+     * @throws Exception
      */
-    public function __construct(ModelManager $modelManager, LoggerInterface $logger)
+    public function __construct(ModelManager $modelManager, LoggerInterface $logger, DIContainer $container)
```

Comment:
i know this is werid, but can you remove the type "DIContainer"
i added that once and it failed in a shop within shopware...it was another incompatible class that was used
you can keep it in line 45 as annotation, but just remove it in the constructor, thx

Reply:
Ok, removed it.